lib/review/pdfmaker.rb

```diff
@@ -486,6 +494,8 @@ def template_content
         template_path = 'layouts/layout.tex.erb'
       end
       ReVIEW::Template.generate(path: template_path, mode: '-', binding: binding, template_dir: template_dir)
+    rescue => e
+      error! "template or configuration error: #{e.full_message}"
```

Comment:
```suggestion
      error! "template or configuration error: #{e.full_message(highlight: false)}"
```

いちおうこれでハイライトを抑制しておいた方が良さそうでした。